components/CreateActivity/CreateActivityForm.js

```diff
@@ -58,7 +91,11 @@ export default function ActivityForm() {
           Title
           <StyledInput type="text" id="title" name="title" required />
         </StyledLabel>
-
+   <StyledLabel htmlFor="picture">Picture
+        <ImageUpload
+          key={formKey}
+          onFileSelect={(file) => setSelectedFile(file)}
+        /></StyledLabel>
```

Comment:
your code is (again) not formatted correctly. I would suggest you disable the autosave feature so you don't forget to hit cmd+s and therefore the the auto formatting from prettier.